pkgs/development/python-modules/snug/default.nix

```diff
@@ -0,0 +1,35 @@
+{ buildPythonPackage, lib, fetchFromGitHub, glibcLocales
+, pytest, pytest-mock, gentools
+, typing, singledispatch, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "snug";
+  version = "1.3.4";
+
+  # Pypi doesn't ship the tests, so we fetch directly from GitHub
+  src = fetchFromGitHub {
+    owner = "ariebovenberg";
+    repo = "snug";
+    rev = "v${version}";
+    sha256 = "0jmg0sivz9ljazlnsrrqaizrb3r7asy5pa0dj3idx49gbig4589i";
+  };
+
+  buildInputs = [ glibcLocales ];
+  LC_ALL = "en_US.UTF-8";
```

Comment:
You're sure this is necessary? It builds fine for me without.

Reply:
It certainly was when I was fetching from Pypi. I will take a look

Reply:
Without it, I still get
```
  File "setup.py", line 21, in <module>
    long_description=read('README.rst') + '\n\n' + read('HISTORY.rst'),
  File "setup.py", line 9, in read
    return rfile.read()
  File "/nix/store/yk7hci7pg3n2kyjh8il317wvd5nam3aw-python3-3.6.7/lib/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xf0 in position 5: ordinal not in range(128)
```

Reply:
That's strange: Without the locale setting, `python37.pkgs.snug` builds fine while `python36.pkgs.snug` doesn't. Let's just keep it in.